package.json

```diff
@@ -54,7 +54,7 @@
     "remark-lint": "^9.1.0",
     "remark-preset-lint-recommended": "^6.1.1",
     "rimraf": "^3.0.2",
-    "stylelint": "^15.10.1",
+    "stylelint": "^14.16.1",
```

Comment:
Is it possible to use v15 or does that break the stencil lint?

Reply:
Anything above v14 introduces issues with peer dependencies. Stenci-community requires <v15 otherwise we end up with a situation like the design-system where we need to add --legacy-peer-deps everytime we do an install.